README.md

```diff
@@ -71,22 +71,21 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+This code would make most sense as a model method. We would want to keep this code out of the controller to keep the controller clean. In the model we would be able to write out a method concerning song ratings/colors.
 ```
 
```

Comment:
We didn't talk about these much, but the answer would probably make the most sense as a helper method because because these are intended to "help" render HTML based on conditions. Controller methods concern how data moves between routes. Model methods concern how data goes to and comes from the database.